build/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM registry.access.redhat.com/ubi7/ubi-minimal:latest
```

Comment:
Can this be set to custom image as done here: https://github.com/openebs/node-disk-manager/blob/master/build/ndm-operator/Dockerfile.in